content/en/observability_pipelines/split_logs/http_client.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: Split Logs for HTTP Client
 disable_toc: false
+aliases:
```

Comment:
I think you might've accidentally added this alias (it's a circular redirect that points to itself/the same file).

Reply:
Yes, you're right, thanks for catching that! I'll remove